arrays/arrays.c

```diff
@@ -40,13 +40,13 @@ Array *create_array (int capacity) {
 void destroy_array(Array *arr) {
 
   // Free all elements
```

Comment:
you want to free each element, you are probably going to want to loop through the elements